src/app/password-bars/password-bars.component.scss

```
@@ -0,0 +1,25 @@
+.strength-blocks {
+  display: flex;
+  gap: 5px;
+  margin-top: 10px;
+
+  &__block {
+    width: 33%;
```

```suggestion
    flex: 1;
```
it's better bc number of block can change